data.sql

```diff
@@ -40,3 +40,24 @@ UPDATE animals SET owner_id = CASE
   ELSE owner_id
 END;
 
```

Comment:
Great job @Grandi0z 🤝

- [ ] As stated in the project requirements, kindly add data for the `vets` table, `specialization` table, and `visits` table to the `data.sql` file.
- [ ] As stated in the project requirements, kindly create queries to answer questions in
transactions. Kindly add these queries to the `queries.sql` file.